command/pr.go

```diff
@@ -99,7 +99,7 @@ func prStatus(cmd *cobra.Command, args []string) error {
 
 	printHeader(out, "Current branch")
 	if prPayload.CurrentPRs != nil {
-		printPrs(out, 0, prPayload.CurrentPRs...)
+		printPrs(out, 0, prPayload.CurrentPRs[0]) // Assume it is the most recent PR for the current branch
```

Comment:
📝 This assumption is based on [orderBy: { field: CREATED_AT, direction: DESC }](https://github.com/doi-t/cli/blob/7a0a665/api/queries_pr.go#L191) for `pullRequests`.

Reply:
This feels slightly risky to me. If `CurrentPRs` is a non-null empty slice, this line will panic.

Since it makes no sense to even return multiple records here in `prPayload`, could you instead backtrack the change that introduced `CurrentPRs` and simply return a field `CurrentPR` that can either be null or is a pointer to the latest PR belonging to the current branch?